input/otlp/metrics.go

```diff
@@ -186,6 +186,55 @@ func (c *Consumer) handleScopeMetrics(
 						event.DataStream = modelpb.DataStreamFromVTPool()
 					}
 					event.DataStream.Namespace = v.Str()
+
+				// The below fields are required by the Processes tab of the
+				// curated Kibana's hostmetrics UI. These fields are
+				// produced by opentelemetry-lib. The below fields are
+				// added to the remapped OTel metrics datapoints as attributes
+				// and are not OTel semconv fields.
+				case "system.process.cpu.start_time":
+					if event.System == nil {
+						event.System = modelpb.SystemFromVTPool()
+					}
+					if event.System.Process == nil {
+						event.System.Process = modelpb.SystemProcessFromVTPool()
+					}
+					if event.System.Process.Cpu == nil {
+						event.System.Process.Cpu = modelpb.SystemProcessCPUFromVTPool()
+					}
+					event.System.Process.Cpu.StartTime = v.Str()
+
+				// `system.process.cmdline` is same as the ECS field `process.command_line`
+				// however, Kibana curated UIs requires this field to work. In addition,
+				// the current Kibana code will not work if this field is added to documents
+				// with `system.process.memory.rss.pct` and other metrics required in the
+				// Processes tab of the Kibana hostmetrics UI. Due to this, we have to process
+				// the datapoint field added by the opentelemetry-lib instead of directly
+				// processing the OTel semconv resource attribute `process.command_line`.
```

Comment:
@felixbarny @axw This means that we would not be able to remove the attributes added at https://github.com/elastic/opentelemetry-lib/blob/54a9569b27e3f11edc0d660dc79ac865256f3cd3/remappers/hostmetrics/process.go#L271-L295 and rely on the translations performed in APM/esexporter without UI fixing such hard dependencies. One such case I have explained [here](https://github.com/elastic/observability-dev/issues/3704#issuecomment-2207351332) (elastic only link).